api/admin/controller/__init__.py

```diff
@@ -2609,7 +2609,15 @@ def validate_formats(self, settings=None):
     def _value(self, field):
         # Extract the user's input for this field. If this is a sitewide setting,
         # then the input needs to be accessed via "value" rather than via the setting's key.
-        return flask.request.form.get(field.get("key")) or flask.request.form.get("value")
+        # We use getlist instead of get so that, if the field is such that the user can input multiple values
+        # (e.g. language codes), we'll extract all the values, not just the first one.
+
+        value = flask.request.form.getlist(field.get("key"))
+        if not value:
+            return flask.request.form.get("value")
```

Comment:
Would it make sense to also call `getlist` here?

Reply:
Unnecessary; this is only called if a sitewide setting is being saved, and none of the sitewide settings accepts multiple values.  (There might be a case to be made for changing the "excluded audiobook sources" setting to make it accept multiple values, but that seems like a conversation for another branch...)